modules/gapi/include/opencv2/gapi/core.hpp

```diff
@@ -9,10 +9,10 @@
 #define OPENCV_GAPI_CORE_HPP
 
 #include <math.h>
-
 #include <utility> // std::tuple
 
 #include <opencv2/imgproc.hpp>
+#include <opencv2/gapi/imgproc.hpp>
```

Comment:
Does this headers make sense for core?

Reply:
Yes, <opencv2/gapi/imgproc.hpp> header is necessary for backward compatibility